### ISSUE TITLE
[a11y [MatSnackBar] Screen reader announces message twice.

### ISSUE THREAD

**Issue by glushkova91** [angular/components#14825]
#### What is the expected behavior?
Screen reader should announce snack bar message only once.

#### What is the current behavior?
Using `open` API method the message is announced twice because:
- role='status' behaves as aria-live and this role is assigned to massage wrapper
- MatSnackBar uses LiveAnnouncer (or smth similar from cdk a11y module) to announce message.

#### What are the steps to reproduce?
1. Open shared example
https://stackblitz.com/angular/krygnaabbpg?file=app%2Fsnack-bar-overview-example.ts
2. Enable SR.
3. Go to "Show Snack-bar" button and press Enter.
4. Observe the result - snack-bar's message is announced twicw

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Chromevox- Google Chrome OS Version 64.0.3282.190

**Comment by glur4k** [comment]
For me it's announced when it's opened and when it's closed (the announcement on close is unnecessary and should be omitted IMHO).

Using ChromeVox and Google Chrome 71.0.3578.98.

**Comment by crisbeto** [comment]
Should be resolved by https://github.com/angular/material2/pull/14504.
